ecf/dev/sub_level2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -login
+
+#module load envvar/1.0
+#module load PrgEnv-intel/8.1.0
+#module load intel/19.1.3.304
+#module load craype
+#module load cray-mpich
+#module load grib_util/1.2.2
+#module load prod_util/2.0.5
+#module load prod_envir/2.0.4
+#module load cray-python
+
+source ~/.bashrc
```

Comment:
what?

Reply:
This is for development test script under ecf/dev. NCO will not use this. But I can remove this if you think it is not necessary.

Reply:
yes. I don't think this script needs to live in a repository.
Instructions on how to use it should live in a README or something, which is what this script is essentially providing.

Reply:
this script is deleted